src/storage/sequelize.ts

```diff
@@ -27,8 +27,13 @@ export type SequelizeType = {
 	isDefined(modelName: string): boolean;
 	model(modelName: string): any;
 	define(modelName: string, columns: {}, options: {}): {};
+	dialect: {
+		name: string;
```

Comment:
```suggestion
	dialect?: {
		name?: string;
```

Make these optional so the other tests won't complain